File: gin-vue-admin-main/server/router/wl_driver/wl_driver_market.go
package wl_driver

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type WlDriverMarketRouter struct{}

func (s *WlDriverMarketRouter) InitWlDriverMarketRouter(Router *gin.RouterGroup, PublicRouter *gin.RouterGroup) {
	wlDriverMarketRouter := Router.Group("wlDriverMarket").Use(middleware.OperationRecord())
	wlDriverMarketRouterWithoutRecord := Router.Group("wlDriverMarket")
	{
		wlDriverMarketRouter.POST("createWlDriverMarket", wlDriverMarketApi.CreateWlDriverMarket)
		wlDriverMarketRouter.DELETE("deleteWlDriverMarket", wlDriverMarketApi.DeleteWlDriverMarket)
		wlDriverMarketRouter.DELETE("deleteWlDriverMarketByIds", wlDriverMarketApi.DeleteWlDriverMarketByIds)
		wlDriverMarketRouter.PUT("updateWlDriverMarket", wlDriverMarketApi.UpdateWlDriverMarket)
	}
	{
		wlDriverMarketRouterWithoutRecord.GET("findWlDriverMarket", wlDriverMarketApi.FindWlDriverMarket)
		wlDriverMarketRouterWithoutRecord.GET("getWlDriverMarketList", wlDriverMarketApi.GetWlDriverMarketList)
	}
}

File: gin-vue-admin-main/web/src/api/wl_driver/wlDriverMarket.js
import service from '@/utils/request'

export const getWlDriverMarketList = (params) => {
  return service({
    url: '/wlDriverMarket/getWlDriverMarketList',
    method: 'get',
    params
  })
}

export const createWlDriverMarket = (data) => {
  return service({
    url: '/wlDriverMarket/createWlDriverMarket',
    method: 'post',
    data
  })
}

export const updateWlDriverMarket = (data) => {
  return service({
    url: '/wlDriverMarket/updateWlDriverMarket',
    method: 'put',
    data
  })
}

export const deleteWlDriverMarket = (params) => {
  return service({
    url: '/wlDriverMarket/deleteWlDriverMarket',
    method: 'delete',
    params
  })
}

export const deleteWlDriverMarketByIds = (params) => {
  return service({
    url: '/wlDriverMarket/deleteWlDriverMarketByIds',
    method: 'delete',
    params
  })
}

export const findWlDriverMarket = (params) => {
  return service({
    url: '/wlDriverMarket/findWlDriverMarket',
    method: 'get',
    params
  })
} 